### DISCUSSION TITLE
Is the current master ready for release? Would like chart.getInitialScaleBounds()

### DISCUSSION THREAD

**Discussion by ozgb** [chartjs/chartjs-plugin-zoom#593] [General]
Looks like the documentation is up to date, and the CI is passing - is there any reason I shouldn't use the head of master in my project?

And further, is there a reason a new release hasn't been made? Thanks!

**Comment by kurkle** [comment]
No reason for not using master. I was hoping to address couple more issues for the next release, but there is no real point for the delay.
